src/odemis/driver/xt_client.py

```diff
@@ -2327,8 +2327,13 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                 self.parent.move_stage(pos, rel=rel)
                 time.sleep(0.1)  # It takes a little while before the stage is being reported as moving
 
+                # Get the target position in absolute coordinates
+                if rel:
+                    target_pos = {ax: val + pos[ax] for ax, val in pos.items()}
```

Comment:
I think this should be `ax: orig_pos[ax] + val` . The current code just doubles the values in pos!